locales/ro_RO/LC_MESSAGES/duckduckgo.po

```diff
@@ -2458,15 +2464,16 @@ msgstr "Altceva"
 
 msgctxt "homepage onboarding"
 msgid "Our privacy policy is simple: we don’t collect or share any of your personal information."
-msgstr "Politica noastră de confidenţialitate este simplă: noi nu colectăm sau distribuim nicio informaţie personală a dumneavoastră."
+msgstr "Politica noastră de confidențialitate este simplă: noi nu colectăm sau distribuim nicio informație personală a dumneavoastră."
 
 msgctxt "showcase_traffic"
 msgid "Over %s Billion anonymous searches."
-msgstr ""
+msgstr "Peste %s Miliarde de căutări anonime."
 
+#, fuzzy
 msgctxt "SERP footer content"
 msgid "Over %s in DuckDuckGo privacy donations."
-msgstr ""
+msgstr "Peste %s în donații către DuckDuckGo pentru confidențialitate."
```

Comment:
Is this about donations made by or towards DuckDuckGo?

Reply:
This refers to donations made by DDG.

Reply:
I see. I'll have to update this string, then. I'll try to do it this evening.